src/linear_algebra/matrix.lean

```diff
@@ -907,25 +907,14 @@ rfl
   reindex_linear_equiv (equiv.refl m) (equiv.refl n) = linear_equiv.refl R _ :=
 linear_equiv.ext $ λ _, rfl
 
-lemma reindex_mul [semiring R]
```

Comment:
I think it's worth it to keep `reindex_mul`, since it is used sometimes and the bundledness of `eₙ` can be useful.

Reply:
The bundled version is now `minor_mul_equiv` - we could have a matching `reindex_mul`, but if we had that we'd probably want to copy all the other lemmas about `minor` too.